src/workflows-app/utils/submission-utils.ts

```diff
@@ -21,6 +23,27 @@ export const AutoRefreshInterval = 1000 * 60; // 1 minute
 export const WdsPollInterval = 1000 * 30; // 30 seconds
 export const CbasPollInterval = 1000 * 30; // 30 seconds
 
+export type InputTableData = {
+  configurationIndex: number;
+  inputTypeStr: string;
+  input_name: string;
+  input_type: InputType;
```

Comment:
Most of Terra UI code uses camel case. Is there a reason why these properties need to be snake case?